tests/graphql/02-root-queries/suite.rc

```diff
@@ -0,0 +1,100 @@
+[meta]
+    title = BAZ
+    description = Some suite baz
+    URL = 'tcp://localhost:<port>/graphql'
+    group = round table
+    importance = critical
+    ranking = one
+
+[cylc]
+    cycle point format = CCYYMMDDThh
+    UTC mode = True
+
+[scheduling]
+    initial cycle point = 20190101T00
+    [[special tasks]]
+        clock-trigger = foo(PT0H), qux(PT0H)
+        sequential = foo, bar, qux, baa, qaz
```

Comment:
I never used it, so had a look at the docs... the documentation states that the sequential declaration is deprecated, in favor of explicit inter-cycle triggers. Is there any chance that we may have to update this test if sequential is removed? If not, feel free to resolve conversation (but good to learn about it)